packages/caravan-clients/src/client.ts

```diff
@@ -36,11 +36,48 @@ export interface UTXO {
   };
 }
 
+export interface Transaction {
```

Comment:
Is this basically the default transaction object returned from bitcoind? If so maybe we can rename this something more explicit like `BitcoindTransaction` (that way it also makes more sense exporting from the clients package).

For structural reasons it also might make sense now to move this to a types.ts file?

Reply:
This is not the default reponse from bitcoind. We tweak response for bitcoind, mempool/blockstream and format it.